dfanalyzer/dftracer.py

```diff
@@ -177,27 +177,6 @@ def get_size(filename):
     return int(size)
 
 
-def get_conditions_default(json_obj):
-    io_cond = "POSIX" == json_obj["cat"]
-    return False, False, io_cond
-
-
-def get_conditions_deepspeed(json_obj):
-    app_cond = "__getitem__" in json_obj["name"] or "checkpoint" in json_obj["name"]
-    io_cond = "POSIX" == json_obj["cat"] or "STDIO" == json_obj["cat"]
-    compute_cond = "compute" in json_obj["name"]
-    return app_cond, compute_cond, io_cond
-
-
-def get_conditions_generic(json_obj: dict):
-    checkpoint_cond = any(
-        any(cond == json_obj[prop] for cond in conditions) for prop, conditions in COND_CHECKPOINT.items()
-    )
-    compute_cond = any(any(cond == json_obj[prop] for cond in conditions) for prop, conditions in COND_COMPUTE.items())
-    read_cond = any(any(cond == json_obj[prop] for cond in conditions) for prop, conditions in COND_READ.items())
-    return checkpoint_cond, compute_cond, read_cond
-
-
```

Comment:
@izzet and @hariharan-devarajan 

I noticed that we remove the "conditions_fn" now. Is this intended?

Reply:
Yes, no need for this anymore

Reply:
This functionality is now covered with the layer definitions = `analyzer.preset.layer_defs`